job-dsls/jobs/prod/prod_create_images_templates_and_sources.groovy

```diff
@@ -0,0 +1,31 @@
+/**
+* Create the OpenShift images templates and sources
+*/
+def scriptTemplate = this.getClass().getResource("job-scripts/prod_create_images_templates_and_sources.jenkinsfile").text
+def parsedScript = scriptTemplate.replaceAll(/<%=\s*(\w+)\s*%>/) { config[it[1]] ?: '' }
+
+def folderPath = "PROD"
+folder(folderPath)
+
+pipelineJob("${folderPath}/create-images-templates-and-sources") {
+    description('This job creates the Openshift images templates and sources.')
```

Comment:
```suggestion
    description('This job creates the Openshift images templates and sources, it must be run separately for RHPAM and RHDM.')
```

I think it would be good to mention it as it's otherwise unclear if you aren't familiar with the script.